go.mod

```diff
@@ -455,3 +468,7 @@ replace github.com/microcosm-cc/bluemonday => github.com/microcosm-cc/bluemonday
 //
 // Then, run `make modtidy` in this repository.
 // This ensures that go.mod and go.sum are up-to-date.
+
+replace github.com/dapr/components-contrib => ../components-contrib
```

Comment:
!! I know you are aware, but just leaving this comment here, so that the we don't forget to replace this.